dialogy/plugins/text/list_search_plugin/__init__.py

```diff
@@ -307,14 +307,16 @@ def get_entities(self, transcripts: List[str], lang: str) -> List[BaseEntity]:
         entities: List[BaseEntity] = []
 
         for i, matches_on_transcript in enumerate(matches_on_transcripts):
-            for text, label, value, span, Score in matches_on_transcript:
+            for text, label, value, span, score in matches_on_transcript:
+                if score == float(0):
```

Comment:
`int(score)` is better than `float(0)`

Reply:
if we do `int(score)` then it would always be 0, since `0<=score<=1`

Reply:
then let's use 0.0 for score? also that should still equate if we were using a 0 too.